lib/whitehall/announcement_filter_option.rb

```diff
@@ -1,12 +0,0 @@
-module Whitehall
```

Comment:
Is there any database cleanup we can do as a result of this?

Reply:
I don't think so.  None of this was committed to the database, it seemed to just be held in memory whilst the application ran.